shell/platform/fuchsia/flutter/platform_view.cc

```diff
@@ -937,4 +942,105 @@ flutter::PointerDataDispatcherMaker PlatformView::GetDispatcherMaker() {
   };
 }
 
+// Ad hoc pretty printer.
+static std::ostream& operator<<(
```

Comment:
Nit:  Please move these into the anonymous namespace {} at the top of the file, to aid readability.  These might even deserve their own file